singer-io/tap-hubspot
# TDL 16686 Implement request timeout

# Description of change
- Added request timeout for API requests with default timeout 300 seconds

# Manual QA steps
 - Set small timeout and verify that sync calls are backing off for timeout error.
 - Provided timeout in integer, float, and string format and verified that it's used in the request.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
